pkgs/tools/admin/auth0-cli/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "auth0-cli";
+  version = "0.11.10";
+
+  src = fetchFromGitHub {
+    owner = "auth0";
+    repo = "auth0-cli";
+    rev = "v${version}";
+    hash = "sha256-1/T2hpSNamorWFuaSBoLsGpe9I06HGew9S3yJsDLmLQ=";
+  };
+
+  vendorHash = "sha256-d9ZwK/LAZGgeagGsg3bGYnVykfQcCLUex0pe/PUCtkA=";
+
```

Comment:
Need to set `subPackages` to avoid building this docs tool
https://github.com/auth0/auth0-cli/blob/main/cmd/build_doc/main.go#L1


(might want to then unset it to actually run all the tests, see pkgs/applications/networking/cluster/starboard/default.nix)

Reply:
Fixed in the preCheck